### ISSUE TITLE
Dynamically update the tabindex attribute instead of applying tabindex=1 to all cells

### ISSUE THREAD

**Issue by payten** [sakaiproject/sakai#1810]
Originally raised in #1789 from an accessibility audit conducted on the GradebookNG tool:

> The cells themselves are all in the tab order due to tabindex="0" attributes. To unclutter the taborder, the value should be changed to -1. The focusable elements inside the cells can still be in the tab order, and the cells themselves can be reached through arrow key nav. In order to keep the grid itself part of the tab order, the currently active (i.e. highlighted) cell should always have a tabindex value of 0. This means that as keyboard focus is given or removed from a cell the cell's tabindex value should be updated (to "0" and "-1" respectively).

Such a change will require a large refactor of the current keyboard navigation JavaScript and wasn't tackled in the previous attempt to satisfy all audit recommendations.

Perhaps an alternative would be to offer a quick way for screenreader users to jump over the table or jump to the beginning of the table via `sr-only` links?

**Comment by jeffpasch** [comment]
Hi @payten . Should we hold off on this until we pull in the refactored table work?
